RxJavaTemplate/app/build.gradle.kts

```diff
@@ -0,0 +1,129 @@
+            keyAlias = keystoreProperties.getProperty("KEY_ALIAS") as String
+        }
+    }
+
+    compileSdk = Versions.ANDROID_COMPILE_SDK_VERSION
+    defaultConfig {
+        applicationId = "co.nimblehq.rxjava"
+        minSdk = Versions.ANDROID_MIN_SDK_VERSION
+        targetSdk = Versions.ANDROID_TARGET_SDK_VERSION
+        versionCode = Versions.ANDROID_VERSION_CODE
+        versionName = Versions.ANDROID_VERSION_NAME
+        multiDexEnabled = true
+        vectorDrawables.useSupportLibrary = true
+        testInstrumentationRunner = "co.nimblehq.rxjava.TestRunner"
+    }
+
+    buildTypes {
+        getByName(BuildType.RELEASE) {
+            isMinifyEnabled = true
+            isDebuggable = false
+            isShrinkResources = true
+            proguardFiles(
+                getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro"
+            )
+            signingConfig = signingConfigs["config"]
+        }
+        getByName(BuildType.DEBUG) {
+            // For quickly testing build with proguard, enable this
+            isMinifyEnabled = false
+        }
+    }
+
+    flavorDimensions(Flavor.DIMENSIONS)
+    productFlavors {
+        create(Flavor.STAGING) {
+            applicationIdSuffix = ".staging"
+        }
+
+        create(Flavor.PRODUCTION) {}
+    }
+
+    sourceSets["test"].resources {
+        srcDir("src/test/resources")
+    }
+
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+    }
+
+    kotlinOptions {
+        jvmTarget = JavaVersion.VERSION_1_8.toString()
+    }
+
+    packagingOptions {
+        exclude("META-INF/DEPENDENCIES.txt")
+        exclude("META-INF/LICENSE.txt")
+        exclude("META-INF/NOTICE.txt")
+        exclude("META-INF/NOTICE")
+        exclude("META-INF/MANIFEST.MF")
+        exclude("META-INF/MANIFEST.mf")
+        exclude("META-INF/LICENSE")
+        exclude("META-INF/DEPENDENCIES")
+        exclude("META-INF/notice.txt")
+        exclude("META-INF/license.txt")
+        exclude("META-INF/dependencies.txt")
+        exclude("META-INF/services/javax.annotation.processing.Processor")
+        exclude("META-INF/rxjava.properties")
+    }
+
+    testOptions {
+        unitTests {
+            // Robolectric resource processing/loading
+            isIncludeAndroidResources = true
+            isReturnDefaultValues = true
+        }
+    }
+
+    tasks.withType<Test> {
+        configure<JacocoTaskExtension> {
+            isIncludeNoLocationClasses = true
+        }
+    }
+
+    buildFeatures {
+        viewBinding = true
+    }
+}
+
+dependencies {
+    implementation(project(Module.COMMON_RX))
+    implementation(project(Module.DOMAIN))
+
+    implementation(fileTree(mapOf("dir" to "libs", "include" to listOf("*.jar"))))
+
+    kapt(Dependencies.AppModule.kapts)
+
+    implementation(Dependencies.AppModule.implementations)
```

Comment:
Would it be better if we could reduce it to `AppModule.implementations` ? 🙏 

But keep the class named as `Dependencies.kt`